packages/react-sdk/docusaurus/docs/React/02-tutorials/02-audio-room.mdx

```diff
@@ -3,4 +3,543 @@ title: Audio Room Tutorial
+2. `StreamCallProvider`: it is initialized with a `Call` and provides us with a lot of useful and convenient hooks to have easy access to the necessary call parameters
+
+:::tip
+To learn more about the `MediaDevicesProvider` you should check out the [Camera & Microphone](../../guides/camera-and-microphone) guide.
+:::
+
+So, if we follow that structure and have a component ready for showing the room that is currently active called `RoomActive` we can render it like this:
+
+```tsx
+<MediaDevicesProvider initialAudioEnabled={false} initialVideoEnabled={false}>
+  <StreamCallProvider call={call}>
+    <RoomActive />
+  </StreamCallProvider>
+</MediaDevicesProvider>
+```
+
+That gives us everything we need to jump into the component that does the real call rendering.
+
+### Building up the UI for a room
+
+Again, we don't want to focus too much on the UI part of it but instead show the concepts of getting access to the necessary data.
+And just as a reminder, you can still take a look at the final project [here](TODO: link to project) to get inspiration.
+
+Inside our `RoomActive` component, we need access to a few things. Luckily, the Stream Video SDK exposes hooks for many of
+them (which is why we needed to wrap the component in a `StreamVideoProvider`).
+
+We'll use the following hooks inside our component:
+
+```tsx title="RoomActive.tsx"
+const call = useCall();
+const participants = useParticipants();
+const { stopPublishingAudio } = useMediaDevices();
+const canSendAudio = useHasPermissions(OwnCapability.SEND_AUDIO);
+```
+
+This shows quite a few hooks, so let's quickly go through them:
+
+1. `useCall()`: it gives us access to the call we created in the previous chapters
+2. `useParticipants()`: we want to render the list of people in the call, so it's useful to have direct access to them here
+3. `useMediaDevices()`: having access to the audio track allows for easy muting of audio
+4. `useHasPermissions(OwnCapability.SEND_AUDIO)`: not everyone will have permission to publish their audio track initially; we can check this and allow them to request permission
+
+You can find the entire code for the component [here](TODO: link to RoomActive file). We want to focus on the usage of the components.
+
+First, to render the participants of the call we can use the `.map` function to go through and render an image and their name:
+
+```tsx
+<div>
+  {participants.map((participant) => (
+    <div key={participant.userId}>
+      <img src={participant.image} alt={`Profile of ${participant.name}`} />
+      <span>{participant.name}</span>
+    </div>
+  ))}
+</div>
+```
+
+This will render the list of participants. If we want to render the speakers, we need to wrap them with the `Audio` element.
+It is required to allow for listening to their audio stream.
+
+Here's how to use it:
+
+```tsx
+<div>
+  {speakers.map((speaker) => (
+    <div key={speaker.userId}>
+      <Audio muted={false} audioStream={speaker.audioStream} />
```

Comment:
Can't we use the `ParticipantView` here? I think that is a higher level component, but I'm not sure it would work for the use-case